moesifwsgi/moesif_data_holder.py

```diff
@@ -4,15 +4,15 @@
 
 class DataHolder(object):
     """Capture the data for a request-response."""
-    def __init__(self, disable_capture_transaction_id, id, method, url, ip, user_id, company_id, metadata, session_token, request_headers, content_length, request_body, transfer_encoding):
+    def __init__(self, disable_capture_transaction_id, id, method, url, ip, request_headers, content_length, request_body, transfer_encoding):
         self.request_id = id
         self.method = method
         self.url = url
         self.ip_address = ip
-        self.user_id = user_id
-        self.company_id = company_id
-        self.metadata = metadata
-        self.session_token = session_token
+        self.user_id = None
```

Comment:
Why the syntax has been changed?

Reply:
Because we decide to remove passing the attribute and set it after sending the response. But let me know if we want to change it back to assign twice.